Automattic/jetpack
# Sharing: deprecate Skype sharing button

Fixes #31624

## Proposed changes:

The button is no longer supported by Skype, we can consequently deprecate it and allow site owners to remove it from their site.

This PR also makes a small change to the message we display for deprecated buttons. Instead of saying "service has shut down", we're a bit more specific, since some services (like Skype) may still be available while their sharing button is no longer supported.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a site running `trunk` or the latest stable version of Jetpack.
* Go to Jetpack > Settings > Social and enable sharing buttons.
* Go to Settings > Sharing and add a Skype sharing button to your site.
* Check your site's frontend; the button should appear.
* Now switch to this branch.
* Check your site's frontend again
    * the button will appear with a message when you're logged in
    * the button will not appear when you're logged out.
